repos/system_upgrade/common/actors/checklvm/libraries/checklvm.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import (
+    CopyFile,
+    DistributionSignedRPM,
+    DracutModule,
+    LVMConfig,
+    TargetUserSpaceUpgradeTasks,
+    UpgradeInitramfsTasks
+)
+
+LVM_CONFIG_PATH = '/etc/lvm/lvm.conf'
+LVM_DEVICES_FILE_PATH_PREFIX = '/etc/lvm/devices'
+
+
+def _report_filter_detection():
+    title = 'LVM filter definition detected.'
+    summary = (
+        'RHEL 9 and above uses the LVM devices file by default to select devices used by LVM. '
+        f'Since this system has LVM filter defined in the {LVM_CONFIG_PATH}, it will be '
+        'used after the upgrade as well.'
```

Comment:
My suggestion (emphasize that we wont modify the config):

'Beginning with RHEL 9, LVM devicesfile is used by default to select devices used by LVM. Since leapp detected the use of LVM filter in the {LVM_CONFIG_PATH} configuration file, the configuration won't be modified to use devicesfile during the upgrade and the LVM filter will remain in use.'